lib/tasks/fix_coord_mismatch.rake

```diff
@@ -7,22 +7,31 @@ task fix_coord_mismatch: :environment do
 
   if by_address.present?
     by_address.each do |result|
-      r = result
-      # checks first addresss
-      geocoded = Geocoder.coordinates("#{r.address}, #{r.city}, #{r.state}, #{r.zip}, #{r.country}")
+      address = parse_add(result, result.address)
+      address2 = parse_add(result, result.address_2)
+      address3 = parse_add(result, result.address_3)
+      geocoded = Geocoder.coordinates(address)
+      geocoded2 = Geocoder.coordinates(address2)
+      geocoded3 = Geocoder.coordinates(address3)
       if geocoded.present?
-        # if present will update record, else will check address_2 and adress_3 fields
```

Comment:
It looks like the logic here changed up a bit. It looks like _before_ if geocoded.present?, it would do an update, otherwise it would update geocoded2 AND geocoded3 if present?

Now there's a few more conditionals and the logic is different. I just want to check that this new is correct